app/digitizingcontroller.cpp

```diff
@@ -77,7 +77,7 @@ bool DigitizingController::isPairValid( QgsQuickFeatureLayerPair pair ) const
   return pair.isValid() && hasEnoughPoints();
 }
 
-void DigitizingController::fixZ( QgsPoint *point ) const
+void DigitizingController::fixZ( std::unique_ptr<QgsPoint> &point ) const
```

Comment:
can we simplify this to `QgsPoint &point` - there's no change of ownership so best to just pass a non-const reference